pages/apis/api_differences.md.erb

```diff
@@ -0,0 +1,48 @@
+# API Differences Between REST and GraphQL
+
+Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
+
+{:toc}
+
+The strengths of the GraphQL API are in complex data queries, and the strengths of the REST API are in creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Features only available in the REST API
+
+* **ANNOTATIONS:** [retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [buildkite-agent annotate command](/docs/agent/v3/cli-pipeline) from within a job).
+* **BUILDS:** set [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
+* **BUILDS:** [get a build by ID](/docs/apis/rest-api/builds#get-a-build).
```

Comment:
A build can be looked up via uuid in graphql:

https://github.com/buildkite/buildkite/blob/15812ebe5aabfeecda41a9fe125604b7420d6ac9/app/models/api/graph/types/query_type.rb#L65